src/bin/cratesfyi.rs

```diff
@@ -584,7 +585,9 @@ impl Context {
     fn storage(&self) -> Result<Arc<Storage>, Error> {
         Ok(self
             .storage
-            .get_or_try_init::<_, Error>(|| Ok(Arc::new(Storage::new(self.pool()?))))?
```

Comment:
Is there a reason to have `Context`'s members all be in `Arc`s instead of it?

Reply:
See https://github.com/rust-lang/docs.rs/pull/911#discussion_r460587484